### ISSUE TITLE
Create Basic Frontend for Scorekeepers

### ISSUE THREAD

**Issue by jebidiah252** [jak103/powerscore#4]
We need to create a front end app that scorekeepers can use. This frontend will be a Vue project that we can run in a docker container for development purposes.

In terms of basic functionality, this UI should be able to:

- Update the current team names
- Start and stop a timer
- Increase/decrease the current period
- Increase/decrease the scores for each team
- Add players to the penalty sections
- Increase/decrease the shots on goal for each team

**Comment by clickclackclark** [comment]
This is something I would be interested in working on.

**Comment by BoyceWhite** [comment]
I'm also happy to help on this issue

**Comment by clickclackclark** [comment]
My plan for this (as of now) is to implement the main display READ functionality which will load out and display dummy data just to get things rolling. Hoping to have that ready by Tuesday. I'm thinking once there is a basic UI template for viewing the data we can work on the other CRUD operations which will likely take a bit longer. That way I'm not preventing others from working on parts of it too if desired. @jebidiah252 Thoughts?

**Comment by clickclackclark** [comment]
@jak103 when you get a moment, would you be able to point us in the right direction for the scorekeeper requirements? I think a few of us were a little confused as to what the functionality of the iPad should be after the demo yesterday. The current proof of concept is modeled after the first comment on this issue to give you an idea of where our understanding of the task was at. Here are some thoughts from our standup meeting.

Does the scorekeeping frontend require any actions? 
* If so, what are they?
* If not, is the purpose of the UI just to display the current game information on the tablet? Should the actions be handled on the backend while connected to the hardware at the game instead? 

Hopefully that makes sense, let me know if I can clarify anything. Thanks for your time!

**Comment by jak103** [comment]
Few comments here in the (parens):

- Update the current team names (This shouldn't need to be done. When the game is created it will have the team names)
- Start and stop a timer (This will be pulled from the timer in the rink. The score keeper frontend shouldn't have this feature)
- Increase/decrease the current period (I'd like to pull this from the rink system as well)
- Increase/decrease the scores for each team (This also should come from the rink system)
- Add players to the penalty sections (The rink system will create the penalties, this frontend needs to be able to associate the penalty with a player)
- Increase/decrease the shots on goal for each team (This pulls from the rink system.)

So basically, everything should pull from the rink system. When there is a goal or a penalty, powerscore should detect it and pop up a place for the score keeper to enter details. For goals, those are the scorer and assisters. For the penalties it is the offender and the associated penalty (tripping, slashing, etc)

**Comment by BoyceWhite** [comment]
@jak103 just want to clarify, I thought we wanted to allow the scorekeepers to score from both the rink system and also a web based scorekeeper UI?
